src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -213,25 +127,13 @@
     "pypi-kenlm": ("kenlm",),
     "pysocks": ("socks",),
     "pytest": ("pytest", "_pytest"),
-    "python-dateutil": ("dateutil",),
-    "python-docx": ("docx",),
-    "python-dotenv": ("dotenv",),
-    "python-editor": ("editor",),
-    "python-engineio": ("engineio",),
-    "python-hcl2": ("hcl2",),
-    "python-jose": ("jose",),
-    "python-json-logger": ("pythonjsonlogger",),
-    "python-levenshtein": ("Levenshtein",),
-    "python-lsp-jsonrpc": ("pylsp_jsonrpc",),
-    "python-magic": ("magic",),
-    "python-pptx": ("pptx",),
-    "python-slugify": ("slugify",),
-    "python-socketio": ("socketio",),
-    "python-statsd": ("statsd",),
     "pyyaml": ("yaml",),
     "pymongo": ("bson", "gridfs", "pymongo"),
     "pymupdf": ("fitz",),
     "pytest-runner": ("ptr",),
+    "python-json-logger": ("pythonjsonlogger",),
+    "python-levenshtein": ("Levenshtein",),
+    "python-lsp-jsonrpc": ("pylsp_jsonrpc",),
```

Comment:
**comment:** added these back in because they don't follow the pattern, but they moved positions.
opted to not extract these out to a pattern.